examples/OcclusionSaliency.ipynb

```diff
@@ -90,7 +80,7 @@
     "test_image_filename = 'catdog.jpg'\n",
     "plt.figure(figsize=(12, 8))\n",
     "plt.axis('off')\n",
-    "plt.imshow(PIL.Image.open(test_image_filename))"
+    "_ = plt.imshow(PIL.Image.open(test_image_filename))"
```

Comment:
Just out of curiousity- why was this null output change required?

Reply:
While functionally required, I noticed that this will help reduce the diffs made when rerunning the notebook: the return from `plt.imshow` constantly changes (incurring a diff), while the result from plotting is consistent across runs (no diff in image data).

If the last line in a cell has a return, then it is printed: in this case the `plt.imshow` returns an Axes class instance. This object instance `repr()` return includes a memory address, which is of course a different hash nearly every time this is run. By assigning this to a variable, the last line no longer "returns" anything and the instance repr is not printed.

Reply:
Ok, great- thanks for the clarification and that makes a lot of sense. We should feel free to use this trick for future demo notebooks.